Dockerfile

```
@@ -1,2 +1,11 @@
 FROM golang:1.17.0-alpine3.14
 
+ENV PATH /usr/local/git/bin:$PATH
```

```suggestion
ENV PATH=/usr/local/git/bin:$PATH
```

`=`をつけた方が明示的かと思います。
つけなくても動きます。